src/components/CustomersTable.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+
+class CustomersTable extends React.Component {
+	render() {
+		return(
+			<div className="row">
+	      		<div className="col">
+		        	<table className="table customers-table">
+			          	<thead>
+			            	<tr>
+			            		<th>Title</th>
+			            		<th>First name</th>
+			            		<th>Surname</th>
+			            		<th>Email</th>
+			            		<th>Room id</th>
+			            		<th>Check in date</th>
+			            		<th>Check out date</th>
+			            	</tr>
+			         </thead>
+			         <tbody>
+			          		<tr>
```

Comment:
Can you see an opportunity to make things a bit more readable here? Could we extract a component? Looking ahead a bit to next week's homework, it might be useful to have some way of rendering lots of rows using some data 😄